src/flamenco/runtime/tests/fd_exec_instr_test.c

```diff
@@ -1192,4 +1228,8 @@ fd_exec_vm_syscall_test_run( fd_exec_instr_test_runner_t *          runner,
 
   *output = effects;
   return actual_end - (ulong)output_buf;
+
+error:
+  _context_destroy( runner, ctx );
+  return 0;
```

Comment:
Might be worth using a cleanup attribute here: https://github.com/firedancer-io/firedancer/blob/main/CONTRIBUTING.md#83-complex-function-exit

Reply:
As far as I know the cleanup attribute can only receive 1 argument. Not sure if it would be clearer if I have to create a new type to encapsulate `runner` and `ctx`. What do you think @topointon-jump ?

Reply:
I'm not against creating a new type, but it's up to you. The only reason I suggest it is because it may help with memory leaks whilst fuzzing.